src/domain/common/whiteboard/excalidraw/CollaborativeExcalidrawWrapper.tsx

```diff
@@ -104,47 +106,15 @@ const CollaborativeExcalidrawWrapper = forwardRef<ExcalidrawAPIRefValue | null,
       };
     }, [handleScroll]);
 
-    const renderCustomUI = useMemo<ExportOpts['renderCustomUI']>(
```

Comment:
I have also removed this, I think nobody was using it and it was not working well